src/Features/Navigation/CoreMenu.php

```diff
@@ -251,6 +277,18 @@ public function register_post_types() {
 		Screen::register_post_type( 'shop_coupon' );
 	}
 
+	/**
+	 * Register all core post types.
+	 */
+	public function register_taxonomies() {
+		Screen::register_taxonomy( 'product_cat' );
+		Screen::register_taxonomy( 'product_tag' );
+		$product_attribute_taxonomies = wc_get_attribute_taxonomies();
+		foreach ( $product_attribute_taxonomies as $taxonomy ) {
+			Screen::register_taxonomy( 'pa_' . $taxonomy->attribute_name );
+		}
```

Comment:
What do you think about matching the product attribute taxonomies with a pattern instead of registering all of them?

If a store has a LOT of attributes, this could be a potential performance hit - especially if it's ultimately unnecessary.

Reply:
I see what you mean; I'm wondering what are some of the higher numbers we've seen so we can test against scenarios like this.

But for now, I've updated this to test against a simple string test for product taxonomies.  This will not be quite as accurate as getting and registering the taxonomies, but conflicts here should hopefully be edge cases.